bgpd/bgp_vty.c

```diff
@@ -14090,8 +14090,12 @@ static void bgp_show_peer_afi(struct vty *vty, struct peer *p, afi_t afi,
 					: "Withdraw");
 
 		/* Receive prefix count */
-		vty_out(vty, "  %u accepted prefixes\n",
-			p->pcount[afi][safi]);
+		vty_out(vty, "  %u accepted prefixes", p->pcount[afi][safi]);
```

Comment:
you've lost the newline here - won't the word "prefixes" run directly onto the number in the new `vty_out` you've added?

Reply:
do you mean this ?
 " %u accepted, %u sent prefixes"

Reply:
sure, anything that fixes it is fine (with me)